kubernetes/cray-service/values.yaml

```diff
@@ -332,10 +332,12 @@ sqlCluster:
     affinity: {}
 
 
-trsWorker:
```

Comment:
This change is not backwards compatible, a user that was using trsWorker.enabled: true wouldn't work anymore.

Reply:
I've bumped the MAJOR version.